### PR TITLE
Task 5 & 7

### PR DESCRIPTION
**Pull request by iskrypnyk**
# Lectures 5 & 7
## Task CSS3 Transitions and animations/ pseudo-elements (all done)

Update your second homework with:
- Add additional pages such as about us, contact, login pages.
- Make sure all URLs are working nicely changing pages.
- Reuse your CSS from the original page. Use CSS extending capabilities.
- Use BEM naming when you doing so like described here. http://getbem.com/naming/
- You can also split your CSS to few files cause it can get quite big already.
- Try to use transitions when it's needed.
- Make a custom buttons 
- Make custom text input (Also you can read about how to style round buttons and checkboxes, it's very complex task, so keep it as a bonus, it's not required, but read at least about it, you will learn a lot).

 ## Add "target input" and "months to target input" (also done)**
- Take the index.html that is attached to the materials post of Lesson 7 or this homework
- Add two inputs to the HTML
- Get them using selector
- Add targetSum and monthToTarget fields to budget object
- Pass to them as a default values to the correct inputs
- addEventListener with input event and a callback function to this inputs to update our budget object
- change logic of button click callback so when you click calculate it shows not only previous message, but also additional saying will user be able to get to the target sum within selected amount of months or not.

### CODE REVIEW

File: Ihor.Skrypnyk/7/index.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      } = budget
+      if ((income - expenses) * mounthToTarget >= targetSum) {
+
+        targetMessage = "You will reach your target";
+      } else {
+        targetMessage = "You have to work on your income/expenses to reach such target";
+      }
+
+
+      result.innerHTML = `Savings per month: ${income - expenses} ` + "<br>" + targetMessage
```
Comment (RIP21) [inline Ihor.Skrypnyk/7/index.html]: Here you can use template string literal.
```javascript
result.innerHTML = `Savings per month: ${income - expenses} <br>
  ${targetMessage}`
```
Like that.

File: Ihor.Skrypnyk/5/html/index.html [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        <input type="password" placeholder="Enter Password" name="psw">
+        <button class="modal-button" type="submit">Login</button>
+        <input type="checkbox" style="margin:26px 30px;"> Remember me
+        <a href="#" style="text-decoration:none; float:right; margin-right:34px; margin-top:26px;">Forgot Password ?</a>
+      </div>
+
+    </form>
+  </div>
+  <script>
+    let modal = document.getElementById('modal-wrapper');
```
Comment (RIP21) [inline Ihor.Skrypnyk/5/html/index.html]: Safer is to use querySelector :) It's a new API, so it's working the same way between browsers, when getElementById is old, so can have some stupid bugs.

Comment (RIP21) [reply Ihor.Skrypnyk/5/html/index.html]: Also, here you don't reassign elements, so use const. You just mutating the properties of object, while object is still the same.

File: Ihor.Skrypnyk/5/html/index.html [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    let close = document.querySelector('.close');
```
Comment (RIP21) [inline Ihor.Skrypnyk/5/html/index.html]: This is dangerous, cause you can get more than 2 elements with this class name, so it's better to use and id or data-id attribute for more reliable selection.

File: Ihor.Skrypnyk/5/html/contactUs.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        <input type="text" placeholder="Enter Username" name="uname">
+        <input type="password" placeholder="Enter Password" name="psw">
+        <button class="modal-button" type="submit">Login</button>
+        <input type="checkbox" style="margin:26px 30px;"> Remember me
+        <a href="#" style="text-decoration:none; float:right; margin-right:34px; margin-top:26px;">Forgot Password ?</a>
+      </div>
+
+    </form>
+  </div>
+  <script>
```
Comment (RIP21) [inline Ihor.Skrypnyk/5/html/contactUs.html]: Same as previous. Good job tho :)

File: Ihor.Skrypnyk/5/html/index.html (hunk 2 above)
Comment (iskrypnyk) [reply Ihor.Skrypnyk/5/html/index.html]: Done

File: Ihor.Skrypnyk/5/html/contactUs.html [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+      <div class="imgcontainer">
+        <span class="close" title="Close PopUp">&times;</span>
+        <h1 style="text-align:center">Login to the website</h1>
+      </div>
+
+      <div class="modal-container">
+        <input type="text" placeholder="Enter Username" name="uname">
+        <input type="password" placeholder="Enter Password" name="psw">
+        <button class="modal-button" type="submit">Login</button>
+        <input type="checkbox" style="margin:26px 30px;"> Remember me
```
[the remark this answers is not in the document]
Comment (iskrypnyk) [reply Ihor.Skrypnyk/5/html/contactUs.html]: Changed

**Comment by iskrypnyk** [comment]
All changed, please review

File: Ihor.Skrypnyk/7/index.html [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+      if ((income - expenses) * mounthToTarget >= targetSum) {
+
+        targetMessage = "You will reach your target";
+      } else {
+        targetMessage = "You have to work on your income/expenses to reach such target";
+      }
+
+
+      result.innerHTML = `Savings per month: ${income - expenses} <br>
+<<<<<<< HEAD
```
Comment (RIP21) [inline Ihor.Skrypnyk/7/index.html]: Merge conflict :D

File: Ihor.Skrypnyk/5/html/contactUs.html (hunk 5 above)
[the remark this answers is not in the document]
Comment (RIP21) [reply Ihor.Skrypnyk/5/html/contactUs.html]: I still can see it mean it's not :P

**Review by RIP21 — CHANGES_REQUESTED** [review]

**Comment by iskrypnyk** [comment]
@RIP21 , Please review again.

**Comment by RIP21** [comment]
@iskrypnyk it looks like a merge commit, not a commit :) Nothing is added :P

**Pull request merged**